cumulus_library/actions/cleaner.py

```diff
@@ -31,7 +31,7 @@ def _execute_drop_queries(
 
 
 def get_unprotected_stats_view_table(
```

Comment:
This method isn't used anywhere, right? Is it public API?

I didn't think the Library had any public utility methods like this.

Reply:
that should be private, yeah

Reply:
OK I was just trying to figure out what this method was about - searching for it didn't yield any results. But I had read the search results wrong - I thought it wasn't used at all. Nevermind me.